source/lexbor/ports/posix/lexbor/core/memory.c

```diff
@@ -6,28 +6,46 @@
 
 #include "lexbor/core/base.h"
 
+static lexbor_malloc_f malloc_f = malloc;
```

Comment:
@zyc9012 

Naming.

static lexbor_memory_malloc_f lexbor_memory_malloc = malloc;
and so on.